### PR TITLE
Inline Documentation for Order.js and OrderDataTable.js

### PR DESCRIPTION
**Pull request by micahelinor**
- [ ] Inline documentation for functions and some variables in those two files.

### CODE REVIEW

**Review by abaruah117 — CHANGES_REQUESTED** [review]
use /* */ for multiline.

**Review by trev4ev — DISMISSED** [review]
lgtm

File: src/components/OrderDataTable.js [hunk 1]
```diff
@@ -16,8 +16,12 @@ class OrderDataTable extends React.Component {
                   id: "Pick Up Date",
                   Header: "Pick Up Date",
                   accessor: row => `${row.PickUpDate}`,
+                  /*Arrow function that returns true if Date values are
```
Comment (trev4ev) [inline src/components/OrderDataTable.js]: Make your comment look like this

```
/*
line 1 blah blah
blah blah
*/

File: src/components/Orders.js [hunk 2]
```diff
@@ -15,16 +15,8 @@ class Orders extends React.Component {
...
-      "Pick Up Date",
-      "Last",
-      "First",
-      "Order#",
-      "Size/Style",
-      "Item Ordered",
-      "Item Quantity"
-    ];
-
+    //this variable arrayOfObjects converts a nested array to an array of objects
```
Comment (trev4ev) [inline src/components/Orders.js]: this should also be multiline

**Review by trev4ev — CHANGES_REQUESTED** [review]
small changes!

**Comment by trev4ev** [comment]
@micahelinor @jamesli898 please make these changes so this can get merged in

**Review by abaruah117 — DISMISSED** [review]
lgtm but please make trevor's changes(multiline formatting)

File: src/components/OrderDataTable.js (hunk 1 above)
Comment (trev4ev) [reply src/components/OrderDataTable.js]: this change was not fixed

**Review by trev4ev — CHANGES_REQUESTED** [review]
please be sure to fix all changes before requesting a review

File: src/components/OrderDataTable.js [hunk 3]
```diff
@@ -16,8 +16,12 @@ class OrderDataTable extends React.Component {
...
+                  lesser and equal to the Date value that the user inputted.*/
                   filterMethod: (filter, row) => {
                     if (
+                      /*Date.parse method parses a string representation of a date,
```
Comment (abaruah117) [inline src/components/OrderDataTable.js]: this also isn't a proper multiline comment, see trevor's comments.

File: src/components/Orders.js [hunk 4]
```diff
@@ -15,16 +15,8 @@ class Orders extends React.Component {
...
-      "Last",
-      "First",
-      "Order#",
-      "Size/Style",
-      "Item Ordered",
-      "Item Quantity"
-    ];
-
+    /*this variable arrayOfObjects converts a nested array to an array of objects
+    since react table needed an array of objects as input.*/
```
Comment (abaruah117) [inline src/components/Orders.js]: also not fixed.

**Review by abaruah117 — CHANGES_REQUESTED** [review]
please make sure to address ALL comments before requesting a review.

File: src/components/OrderDataTable.js (hunk 1 above)
Comment (abaruah117) [inline src/components/OrderDataTable.js]: again, this is the 3rd iteration that you haven't changed these to proper format, see trevor's comments.

File: src/components/OrderDataTable.js (hunk 3 above)
Comment (abaruah117) [reply src/components/OrderDataTable.js]: again

File: src/components/Orders.js (hunk 4 above)
Comment (abaruah117) [reply src/components/Orders.js]: fix this

**Review by abaruah117 — CHANGES_REQUESTED** [review]
please make sure to read all comments and address all requested changes before requesting a review.

File: src/components/OrderDataTable.js (hunk 1 above)
Comment (micahelinor) [reply src/components/OrderDataTable.js]: got it

File: src/components/Orders.js (hunk 2 above)
Comment (micahelinor) [reply src/components/Orders.js]: got it

File: src/components/Orders.js (hunk 4 above)
Comment (micahelinor) [reply src/components/Orders.js]: goit it

**Review by abaruah117 — DISMISSED** [review]
lgtm

**Comment by micahelinor** [comment]
@trev4ev need to approve

**Comment by micahelinor** [comment]
@trev4ev

**Pull request merged**